dask/dataframe/io/hdf.py

```diff
@@ -8,14 +8,21 @@
 from fsspec.utils import build_name_function, stringify_path
 from tlz import merge
 
-from dask import config, multiprocessing
+from dask import config
 from dask.base import compute_as_if_collection, get_scheduler, tokenize
 from dask.dataframe.core import DataFrame
 from dask.dataframe.io.io import _link, from_map
 from dask.dataframe.io.utils import DataFrameIOFunction
 from dask.delayed import Delayed, delayed
 from dask.utils import get_scheduler_lock
 
+try:
+    from dask import multiprocessing
+except (ImportError, ModuleNotFoundError):
+    mpget = object()
```

Comment:
Any reason `object` was used instead of `None`?

Reply:
Further down there is a check if a scheduler `get` is identical to `mpget`. `dask.base.get_scheduler()` *can* return `None`, and I wanted this check to have the same semantics as before (that is, to always be false unless `mpget is multiprocessing.get`). Happy do differently if you think that reasoning is faulty.